dockerfiles/theia-dev/Dockerfile

```diff
@@ -23,6 +23,14 @@ RUN yarn global add @eclipse-che/theia-generator@0.0.1-1543478099
 # change permissions
 RUN find ${HOME} -exec sh -c "chgrp 0 {}; chmod g+rwX {}" \; 2>log.txt
 
+RUN apk update && \
+    apk add --no-cache \
+    openssh-keygen
+RUN apk add --no-cache openssh
+
+# Set up more file watcher
+RUN echo "fs.inotify.max_user_watches = 524288" > /etc/sysctl.conf
```

Comment:
let's move it after line 16 (so before installing something else) as this instruction may not change at all